### PR TITLE
[CONTENT] Very Specific Leader's Den Gathering Outcomes

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

Some extra specific leader's den outcomes (mostly successes, but some fails) for trait/skill combos.

## Why This Is Good For ClanGen

Help traits and skills feel like they really interact with each other to determine how a character acts.

### CODE REVIEW

**Comment by tobes03** [comment]
Still troubleshooting... index error has prevented me from generating these outcomes

File: resources/dicts/events/leader_den/success/other_clan.json [hunk 1]
```diff
@@ -1125,6 +1381,39 @@
...
+    "m_c": {
+      "trait": ["lonesome"],
+      "skill": ["PROPHET,4"]
+    },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "antagonize",
+    "event_text": "At the Gathering, o_c_n's leader repeatedly provokes and challenges c_n. m_c remains composed, not a whisker out of place, and asks o_c_n's leader to name a place and time for them to settle the score. {PRONOUN/m_c/poss/CAP} self-possession riles o_c_n's leader even more",
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/success/other_clan.json]:
```suggestion
    "event_text": "At the Gathering, o_c_n's leader repeatedly provokes and challenges c_n. m_c remains composed, not a whisker out of place, and asks o_c_n's leader to name a place and time for them to settle the score. {PRONOUN/m_c/poss/CAP} self-possession riles o_c_n's leader even more.",
```

File: resources/dicts/events/leader_den/success/other_clan.json [hunk 2]
```diff
@@ -1125,6 +1381,39 @@
...
+    "m_c": {
+      "trait": ["calm"],
+      "skill": ["FIGHTER,1"]
+    },
+    "player_clan_temper": ["any"],
+    "other_clan_temper": ["any"]
+  },
+  {
+    "interaction_type": "antagonize",
+    "event_text": "At the Gathering, m_c forgoes a traditional speech in favor of retelling the tale of a legendary o_c_n leader... but with {PRONOUN/m_c/poss} own special graphically violent twist. o_c_n warriors cover their apprentices' ears, and even o_c_n's leader looks like they'll have nightmares",
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/success/other_clan.json]:
```suggestion
    "event_text": "At the Gathering, m_c forgoes a traditional speech in favor of retelling the tale of a legendary o_c_n leader... but with {PRONOUN/m_c/poss} own special graphically violent twist. o_c_n warriors cover their apprentices' ears, and even o_c_n's leader looks like they'll have nightmares.",
```

File: resources/dicts/events/leader_den/success/other_clan.json [hunk 3]
```diff
@@ -1219,16 +1508,48 @@
     "player_clan_temper": ["any"],
     "other_clan_temper": ["wary"]
   },
+  {
+    "interaction_type": "appease",
+    "event_text": "o_c_n's leader threatens c_n throughout the Gathering. m_c remains silent until {PRONOUN/m_c/poss} turn to speak, then remarks that no cat in o_c_n really wants war. o_c_n warriors shuffle their paws and avoid their leader's gaze, not wanting to admit that m_c is right",
```
Comment (scribblecrumb) [inline resources/dicts/events/leader_den/success/other_clan.json]:
```suggestion
    "event_text": "o_c_n's leader threatens c_n throughout the Gathering. m_c remains silent until {PRONOUN/m_c/poss} turn to speak, then remarks that no cat in o_c_n really wants war. o_c_n warriors shuffle their paws and avoid their leader's gaze, not wanting to admit that m_c is right.",
```

**Review by scribblecrumb — APPROVED** [review]

**Pull request merged**